base/gfspconfig/config.go

```diff
@@ -295,3 +296,7 @@ type ManagerConfig struct {
 	// EnableBucketMigrateCache is used to enable bucket migrate's bucket cache.
 	EnableBucketMigrateCache bool `comment:"optional"`
 }
+
+type QuotaConfig struct {
+	MonthlyFreeQuota uint64 `comment:"optional"`
```

Comment:
please add this config into https://sp_address/status API, so that other apps (e.g. greenfieldscan/dcellar can know how much monthly free quota is offered by a given SP)